docs/components/inputMapper.base.md

```diff
@@ -0,0 +1,300 @@
+
+This component transforms the gamepad inputs into actions. It is derived from the combination of two grades - the
+[`navigator`](navigator.md) grade, which reads and stores the gamepad inputs in the form of model data, and the
+[`configMaps`](configMaps.md) grade provides a configuration map for the gamepad inputs. The navigation features
+available with this component are limited to the **intra-web page navigation** features. (Use the `inputMapper`
+component for **inter-web page navigation** features)
+
+## Using this grade
+
+For the actions to work, you need to use the `gamepad.inputMapperUtils.content` namespace unless the `gamepad` namespace
+is used. Otherwise, the component will throw errors. The configuration options can be changed by either extending the
+component or by passing your own options. If no options are passed, the defaults are used.
+
+``` javascript
+fluid.registerNamespace("gamepad.inputMapper.base");
+fluid.registerNamespace("gamepad.inputMapperUtils.content");
+
+// Either create an instance of the inputMapper.base component.
+var inputMapperBaseInstanceOne = gamepad.inputMapper.base();
+
+// Otherwise create a custom component using the inputMapper.base grade.
+fluid.defaults("my.inputMapperBase.grade", {
+    gradeNames: ["gamepad.inputMapper.base", "my.CustomGrade"]
+});
+var inputMapperBaseInstanceTwo = my.inputMapperBase.grade();
+```
+
+## Component Options
+
+The following component configuration options are supported:
+
+| Option | Type | Default Value | Description |
+| :---: | :---: | :---: | :--- |
+| `cutoffValue` | `{Number}` | 0.20 | The threshold for the gamepad input value to perform an action. |
+| `scrollInputMultiplier` | `{Number}` | 50 | Multiplication factor for the gamepad input value to calculate the offset from the current scroll position (in pixels). |
+
+These options can be provided to the component in the following ways:
+
+``` javascript
+fluid.registerNamespace("gamepad.inputMapper.base");
+fluid.registerNamespace("gamepad.inputMapperUtils.content");
+
+// Either pass options inside the object as an argument while creating an instance of inputMapper.base component.
+var inputMapperBaseInstanceOne = gamepad.inputMapper.base({
+    cutoffValue: 0.15,
+    scrollInputMultiplier: 35
+});
+
+// Otherwise pass it as default options in a custom component using the inputMapper.base grade.
+fluid.defaults("my.inputMapperBase.grade", {
+    gradeNames: ["gamepad.inputMapper.base"],
+    cutoffValue: 0.10,
+    scrollInputMultiplier: 25
+});
+var inputMapperBaseInstanceTwo = my.inputMapperBase.grade();
+```
+
+You can also pass custom configuration map (in the form of model variables) to specify the action and other associated
+parameters for the gamepad inputs as an argument. For information regarding the configuration map model variables, refer
+to the [Component Options](configMaps.md#component-options) section in the [`configMaps`](configMaps.md) grade document.
```

Comment:
"document" is technically correct, but I usually prefer "documentation" in cases like this.